src/PrestaShopBundle/Resources/config/services/core/form/form_builder.yml

```diff
@@ -285,3 +285,10 @@ services:
     arguments:
       - 'PrestaShopBundle\Form\Admin\Improve\International\Locations\StateType'
       - '@prestashop.core.form.identifiable_object.data_provider.state_form_data_provider'
+
+  prestashop.core.form.identifiable_object.builder.tax_rules_group_form_builder:
```

Comment:
In those cases however (common class for multiple services) we still have to use the name definition

Reply:
Ofc